pyinfra/api/connectors/sshuserclient/client.py

```diff
@@ -79,6 +106,25 @@ def parse_config(self, hostname, initial_cfg=None, ssh_config_file=None):
         host_config = ssh_config.lookup(hostname)
         forward_agent = host_config.get('forwardagent') == 'yes'
 
+        missing_host_key_policy = AskPolicy()
+
+        if 'stricthostkeychecking' in host_config:
+            v = host_config['stricthostkeychecking']
+            if v == 'ask':
+                missing_host_key_policy = AskPolicy()
+            elif v == 'no' or v == 'off':
+                missing_host_key_policy = WarningPolicy()
+            elif v == 'yes':
+                missing_host_key_policy = StrictPolicy()
+            else:
+                raise SSHException(f"Invalid value StrictHostKeyChecking={host_config['stricthostkeychecking']}")
+
+        host_keys_file = None
+        if 'userknownhostsfile' in host_config:
+            host_keys_file = host_config['userknownhostsfile']
+            if host_keys_file.startswith('~'):
+                host_keys_file = getenv('HOME') + host_keys_file[1:]
```

Comment:
```suggestion
            host_keys_file = os.expanduser(host_keys_file)
```

[`os.expanduser`](https://docs.python.org/3/library/os.path.html#os.path.expanduser) is a neater way to do this.

Reply:
Done.